package.json

```
@@ -4,7 +4,7 @@
 	"private": true,
 	"scripts": {
 		"dev": "npm run pyodide:fetch && vite dev --host",
-		"dev:5050": "npm run pyodide:fetch && vite dev --port 5050",
+		"dev:5050": "npm run pyodide:fetch && vite devå --port 5050",
```

```suggestion
		"dev:5050": "npm run pyodide:fetch && vite dev --port 5050",
```

The extra character can be removed.